shub/settings.py

```diff
@@ -668,4 +668,20 @@ def __iter__(self):
     }
 
 if ENABLE_GITLAB_AUTH:  # noqa
-    SOCIAL_AUTH_GITLAB_SCOPE = ["api", "read_user"]
+    if "SOCIAL_AUTH_GITLAB_SCOPE" not in locals():
+        SOCIAL_AUTH_GITLAB_SCOPE=[]
+    SOCIAL_AUTH_GITLAB_SCOPE += get_sregistry_envar_list("SOCIAL_AUTH_GITLAB_SCOPE")
+    SOCIAL_AUTH_GITLAB_SCOPE = list(set(SOCIAL_AUTH_GITLAB_SCOPE))
+
+
+if __name__ == "__main__":
```

Comment:
Please make this a management script instead, this is a bit klunky.

Reply:
done!